java/code/src/com/suse/scc/SCCEndpoints.java

```diff
@@ -152,7 +152,7 @@ public String unscoped(Request request, Response response, HubSCCCredentials cre
      * @param token the token
      * @return return {@link SCCRepositoryJson} object for a Hub custom repository
      */
-    public SCCRepositoryJson buildCustomRepoJson(String label, String hostname, String token) {
+    public static SCCRepositoryJson buildCustomRepoJson(String label, String hostname, String token) {
```

Comment:
I will change this in issv3-scc-endpoints branch. It belongs there. A rebase should remove this